package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "app_scaffold",
+  "version": "1.0.1",
+  "description": "A scaffold for developers to build ZAF v2 apps",
+  "keywords": [
+    "zendesk",
+    "app",
+    "framework"
+  ],
+  "author": "Zendesk",
+  "license": "Apache-2.0",
+  "engines": {
+    "node": ">=6.11.5"
+  },
+  "scripts": {
+    "test": "jest",
+    "watch": "webpack --watch --mode=development",
+    "build:dev": "webpack --mode=development",
+    "build": "webpack -p --mode=production",
+    "lint": "standard",
+    "start": "zcli apps:server ./dist"
+  },
+  "devDependencies": {
+    "@zendeskgarden/css-arrows": "^3.1.1",
+    "@zendeskgarden/css-avatars": "^3.0.3",
+    "@zendeskgarden/css-bedrock": "^7.0.3",
+    "@zendeskgarden/css-buttons": "^6.0.3",
+    "@zendeskgarden/css-callouts": "^3.0.3",
+    "@zendeskgarden/css-forms": "^6.0.5",
+    "@zendeskgarden/css-grid": "^0.1.6",
+    "@zendeskgarden/css-menus": "^7.0.4",
+    "@zendeskgarden/css-modals": "^6.1.2",
+    "@zendeskgarden/css-pagination": "^3.0.3",
+    "@zendeskgarden/css-tables": "^3.0.5",
+    "@zendeskgarden/css-tabs": "^5.0.3",
+    "@zendeskgarden/css-tags": "^4.0.4",
+    "@zendeskgarden/css-tooltips": "^4.0.4",
+    "@zendeskgarden/css-utilities": "^3.0.1",
+    "babel-core": "^6.26.3",
+    "babel-jest": "^23.4.0",
+    "babel-loader": "^7.1.5",
+    "babel-polyfill": "^6.26.0",
+    "babel-preset-env": "^1.7.0",
+    "clean-webpack-plugin": "^0.1.19",
+    "copy-webpack-plugin": "^4.5.2",
+    "css-loader": "^1.0.0",
+    "html-webpack-plugin": "^3.2.0",
+    "jest": "^23.4.0",
+    "mini-css-extract-plugin": "^0.4.1",
+    "postcss-import": "^11.1.0",
+    "postcss-loader": "^2.1.6",
+    "postcss-preset-env": "^5.2.1",
+    "precss": "^3.1.2",
+    "standard": "^11.0.1",
+    "webpack": "^4.16.0",
+    "webpack-cli": "^3.0.8"
+  },
+  "dependencies": {}
+}
```

Comment:
both scaffold will need to have their own package.json because,
- they will be using different set of packages
- they will also need to be download as package using zcli when devs run new command in zcli